android/app/proguard-rules.pro

```diff
@@ -23,6 +23,8 @@
 
 -dontnote **
 
+-keep class host.exp.exponent.notifications.NotificationsScoper { *; }
```

Comment:
I'm torn between leaving this here and suggesting using `@DoNotStrip` annotation. I think using annotation is more common and we wouldn't need to add it to multiple ProGuard rules. What do you think?

Reply:
I'll change it to the annotation ;)